bin/run-pantheon-acceptance-tests.sh

```diff
@@ -0,0 +1,149 @@
+else
+  echo "Terminus rsync plugin already installed"
+fi
+
+echo "Authenticating to terminus:"
+terminus auth:login --machine-token="$ACCEPTANCE_PANTHEON_MACHINE_TOKEN"
+
+# Create multidev environment.
+if ! terminus multidev:list $ACCEPTANCE_PANTHEON_SITE --format=csv | tail -n+2 | grep -q "$ACCEPTANCE_PANTHEON_ENV"; then
+  echo "Creating multidev environment for $ACCEPTANCE_PANTHEON_ENV"
+  terminus multidev:create $ACCEPTANCE_PANTHEON_SITE.dev $ACCEPTANCE_PANTHEON_ENV
+  MULTIDEV_ENV_CREATED=1
+else
+  echo "Multidev for $ACCEPTANCE_PANTHEON_ENV already created."
+  MULTIDEV_ENV_CREATED=0
+fi
+
+echo "Checking for environment lock"
+if terminus remote:wp "$ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV" -- core is-installed; then
+  echo "Is installed, now checking if lock is present..."
+  while [[ $( date +%s ) -lt $(( $( terminus remote:wp $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV option get testbed_lock_timestamp 2>/dev/null ) + $ACCEPTANCE_LOCK_TIMEOUT )) ]]; do
+    echo "Waiting 15 seconds for testbed lock to clear (up to $ACCEPTANCE_LOCK_TIMEOUT seconds)..."
+    sleep 15
+  done
+fi
+
+echo "Ensure SFTP mode:"
+terminus connection:set $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV sftp
+
+echo "Wipe environment:"
+if [ "$MULTIDEV_ENV_CREATED" == 1 ]; then
+  terminus remote:wp $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV -- db reset --yes
+else
+  terminus env:wipe $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV --no-interaction --yes
+fi
+
+echo "Update from upstream:"
+terminus upstream:updates:apply --updatedb --accept-upstream $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV
+
+# TODO Consider uploading a pantheon.yml that defines the PHP version as 7.0.
+
+echo "Install WordPress:"
+terminus remote:wp --quiet $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV -- core install \
+  --title="Testbed for $ACCEPTANCE_PLUGIN_SLUG" \
+  --url="$ACCEPTANCE_PANTHEON_SITEURL" \
+  --admin_user="$ACCEPTANCE_PANTHEON_SITE_USERNAME" \
+  --admin_password="$ACCEPTANCE_PANTHEON_SITE_PASSWORD" \
+  --admin_email="$ACCEPTANCE_PANTHEON_SITE_EMAIL" \
+  --skip-email
+
+terminus remote:wp $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV option set testbed_lock_timestamp $( date +%s )
+
+echo "Upgrading to latest version of WP:"
+# TODO: --version=nightly
+terminus remote:wp $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV -- core update
+
+echo "Upload plugin files:"
+terminus rsync build/ $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV:code/wp-content/plugins/$ACCEPTANCE_PLUGIN_SLUG -- -avz --delete
+
+# TODO: Actually, we can let the WP user activate the plugin instead.
+echo "Activating plugin:"
+terminus remote:wp $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV -- plugin activate $ACCEPTANCE_PLUGIN_SLUG
+
+echo "Unlock environment from HTTP Auth so clients can freely connect"
+terminus lock:disable $ACCEPTANCE_PANTHEON_SITE.$ACCEPTANCE_PANTHEON_ENV
+
+phantomjs --webdriver=8643 --web-security=no &
```

Comment:
There's gotta be a better way to spin up PhantomJS from PHP and to then kill it once Behat is done.